numba_cuda/numba/cuda/cuda_bf16.py

```diff
@@ -0,0 +1,8548 @@
+
+# Automatically generated by Numbast Static Binding Generator
```

Comment:
It would be good if Numbast could provide a bit more information that makes this more reproducible - e.g.:

- Numbast version
- Input files
- Command line used for execution
- CUDA toolkit version
- etc.

Though I realise this should probably be a Numbast feature request - what do you think?

Reply:
Yes I think that's a reasonable ask though. The binding file should contain all information from which the binding itself is generated. Perhaps even including the whole of the config file!

Reply:
Addressed in https://github.com/NVIDIA/numba-cuda/pull/166/commits/079f8978fe48d71c81078339eb0feda1b15d248b